lib/galaxy/config/__init__.py

```diff
@@ -380,7 +379,7 @@ def _process_config(self, kwargs):
         self.oidc_config = kwargs.get("oidc_config_file", self.oidc_config_file)
         self.oidc_backends_config = kwargs.get("oidc_backends_config_file", self.oidc_backends_config_file)
         self.oidc = []
-        self.integrated_tool_panel_config = os.path.join(self.mutable_config_dir, self.integrated_tool_panel_config)
+        self.integrated_tool_panel_config = os.path.join(self.managed_config_dir, self.integrated_tool_panel_config)
```

Comment:
```suggestion
        self.integrated_tool_panel_config = self._in_managed_config_dir(self.integrated_tool_panel_config)
```

Reply:
I will apply this for now, but I plan to get rid of such cases by setting `path_resolves_to` in the schema - there are quite a few of those, and more..

I don't want to move on with those until #9371 is in - otherwise I'd have to add more changes to it, which are not directly related to its main theme, and I want to avoid that and keep it more or less atomic. It's not an urgent PR, but it's holding up a bunch of modifications I have in mind.. Sorry for being such a pest, but could you take a look at it, please?

Reply:
I have to work on training material for the Galaxy Admin Training next week, feel free to have it reviewed by another committer!

Reply:
Thanks! Will do.